.github/workflows/cmake.yml

```diff
@@ -94,16 +98,52 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: recursive
+      - uses: actions/checkout@v4
+        if: env.APACHE_ARROW == 'main'
+        with:
+          path: apache-arrow
+          repository: apache/arrow
```

Comment:
Could you move this just before `name: Configure Apache Arrow` because they are related?

Reply:
fix: [c46b275](https://github.com/groonga/groonga/pull/2459/commits/c46b27506d30747ebcaff49d12a78c822223c52c) Sure.